mediathread/templates/main/collection_add.html

```diff
@@ -16,7 +16,12 @@ <h4 class="text-center pt-3">Add To Collection</h4>
                        <div class="card-body">
                            <h5 class="card-title text-center">Import Media</h5>
                            <p class="card-text">
-                               Install Mediathread’s Google Chrome extension to import video, audio, and images into this course from various sites across the web. You must be using Chrome to install this extension.
+                               Install Mediathread’s Google Chrome
+                               extension to import video, audio, and
+                               images into this course from various
+                               sites across the web. You must be using
+                               a browser in the Chrome family to
+                               install this extension.
```

Comment:
Why not provide the wikipedia link here? Likely folks would find that helpful.

Reply:
good idea